libs/books/feature/src/lib/book-search/book-search.component.html

```diff
@@ -1,6 +1,6 @@
 <form [formGroup]="searchForm" (submit)="searchBooks()">
     <mat-form-field floatLabel="never">
-        <input autoFocus matInput type="search" placeholder="Search for books to add to your reading list" formControlName="term" />
+        <input autoFocus matInput type="search" placeholder="Search for books to add to your reading list" formControlName="term" (ngModelChange)='searchBooks()' />
```

Comment:
Use form builder value changes observable in the component instead of `ngModelChange `